components/admin_console/team_channel_settings/channel/list/channel_list.tsx

```diff
@@ -182,7 +182,12 @@ export default class ChannelList extends React.PureComponent<ChannelListProps, C
             }
 
             if (isArchivedChannel(channel)) {
-                iconToDisplay = <ArchiveIcon className='channel-icon'/>;
+                iconToDisplay = (
+                    <ArchiveIcon
+                        className='channel-icon'
+                        data-testid={`${channel.display_name}-archive-icon`}
```

Comment:
I think the proper is to use `channel.name` since display_name might have "space" on it and I'm not particularly sure if that's valid value.

Reply:
Done!